src/v2/Components/SavedSearchAlert/SavedSearchAlertModel.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  Aggregations,
+  ArtworkFilters,
+} from "../ArtworkFilter/ArtworkFilterContext"
+
+export interface SavedSearchAleftFormValues {
+  name: string
+  email: boolean
+}
+
+export interface SavedSearchAlertFormPropsBase {
+  filters: ArtworkFilters
+  aggregations: Aggregations
+  artistId: string
+  artistName: string
+}
+
+export interface SavedSearchAlertMutationResult {
+  id: string
+}
+
+export interface SavedSearchAlertUserAlertSettings {
+  name: string
+  email?: boolean
```

Comment:
you can make `email` **not as optional**. In Eigen it is now optional for the reason that notification toggles under the feature flag